src/test/java/code401challenges/tree/TreeTest.java

```diff
@@ -32,9 +37,14 @@ public void preOrder() {
 
     @Test
     public void postOrder() {
+        assertArrayEquals(new Integer[]{5,6,10,1,4,3,7},

```

Comment:
Post-order should go: LEFT, RIGHT, SELF. So for the tree you made in your test, it should print in this order: 6, 10, 5, 3, 7, 4, 1.

See this page for a good diagram on all traversals: https://javarevisited.blogspot.com/2016/07/binary-tree-preorder-traversal-in-java-using-recursion-iteration-example.html